packages/gatsby-telemetry/src/index.ts

```diff
@@ -24,35 +29,84 @@ function tick(): void {
     .then(() => setTimeout(tick, interval))
 }
 
-module.exports = {
-  trackFeatureIsUsed: (name: string): void => instance.trackFeatureIsUsed(name),
-  trackCli: (input, tags, opts): void =>
-    instance.captureEvent(input, tags, opts),
-  trackError: (input, tags): void => instance.captureError(input, tags),
-  trackBuildError: (input, tags): void =>
-    instance.captureBuildError(input, tags),
-  setDefaultTags: (tags): void => instance.decorateAll(tags),
-  decorateEvent: (event, tags): void => instance.decorateNextEvent(event, tags),
-  setTelemetryEnabled: (enabled): void => instance.setTelemetryEnabled(enabled),
-  startBackgroundUpdate: (): void => {
-    setTimeout(tick, interval)
-  },
-  isTrackingEnabled: (): boolean => instance.isTrackingEnabled(),
-  aggregateStats: (data): IAggregateStats => instance.aggregateStats(data),
-  addSiteMeasurement: (event, obj): void =>
-    instance.addSiteMeasurement(event, obj),
-  expressMiddleware: function (source: string) {
-    return function (
-      _req: express.Request,
-      _res: express.Response,
-      next
-    ): void {
-      try {
-        instance.trackActivity(`${source}_ACTIVE`)
-      } catch (e) {
-        // ignore
-      }
-      next()
+export function trackFeatureIsUsed(name: string): void {
```

Comment:
Silly question perhaps but why do we expose proxies for these rather than the object itself? Concerned of abuse-by-reference?

Reply:
The idea was to expose a singleton instance. and for reasons that I can't recall we ended up with this

Reply:
`export default instance`? ;)

Yeah it's fine this way. Just wondering about the reasoning behind the pattern.